framework/include/wifi_manager/wifi_manager.h

```diff
@@ -290,6 +290,34 @@ wifi_manager_result_e wifi_manager_get_config(wifi_manager_ap_config_s *config);
  */
 wifi_manager_result_e wifi_manager_remove_config(void);
 
+/**
+ * @brief convert mac address (48bit) to mac address string (FF:FF:FF:FF:FF:FF)
+ *
+ * @param[in]  mac_addr  :  mac address 48bit
```

Comment:
You might need to check this syntax on doxygen.
Doxygen get first param and second one separately and show them on different tabs.
```:``` can cause strange format on doxygen.

Reply:
I have confirmed that: is normally displayed in doxygen.

ex)
Parameters
[in]   mac_addr : mac address 48bit
[out] mac_str    : mac address string